src/pspm_options.m

```diff
@@ -843,53 +844,6 @@
     return
   end
 end
-% 2.21.2 set default marker_chan, if it is a glm struct (only for non-raw data)
```

Comment:
These codes were originally written in `extract_segments`. I moved them here because I feel maybe it is more sorted to leave them in this function since they are dealing with `options` fields. Just for your information.

Reply:
yes but they are not needed any longer.

Reply:
yes but this code is not needed any longer.